lib/feature/browser/bloc/browser_tabs_bloc_event.dart

```diff
@@ -0,0 +1,7 @@
+part of 'browser_tabs_bloc_bloc.dart';
+
+@freezed
+class BrowserTabsBlocEvent with _$BrowserTabsBlocEvent {
+  const factory BrowserTabsBlocEvent.addTab({required Uri uri}) = _AddTab;
+  const factory BrowserTabsBlocEvent.closeAll() = _CloseAll;
```

Comment:
What about event to switch from one tab to another?

Reply:
This thing is not ready yet, sorry. This PR is only about HUD.